### DISCUSSION TITLE
Support Oh-my-zsh

### DISCUSSION THREAD

**Discussion by tungdt996** [ohmyzsh/ohmyzsh#12682] [Help] [answered]
hi everyone, I need support about the functionality of oh-my-zsh. 
I want to turn off the function of highlighting an area when clicking on each command block like this, how can I do that?
![image](https://github.com/user-attachments/assets/6cf163d5-8922-4e0e-8f1e-0203016d185f)

**Comment by carlosala** [comment, marked as answer]
Hi! That has to do with your terminal emulator, for sure. Check their docs or open an issue there.
Thanks!

**Reply by tungdt996** [reply]
Thanks. I will check again.
